maistra/vendor/com_googlesource_chromium_v8/genrule_cmd.bzl

```diff
@@ -1,4 +1,4 @@
n  WEE8_BUILD_ARGS+=\" is_debug=false\"\n  WEE8_BUILD_ARGS+=\" v8_symbol_level=1\"\n  WEE8_BUILD_ARGS+=\" v8_enable_handle_zapping=false\"\nfi\n\n# Clang or not Clang, that is the question.\nWEE8_BUILD_ARGS+=\" is_clang=$$IS_CLANG\"\n# Hack to disable bleeding-edge compiler flags.\nWEE8_BUILD_ARGS+=\" use_xcode_clang=true\"\n# Use local toolchain.\nWEE8_BUILD_ARGS+=\" custom_toolchain=\\\"//build/toolchain/linux/unbundle:default\\\"\"\n# Use local stdlibc++ / libc++.\nWEE8_BUILD_ARGS+=\" use_custom_libcxx=false\"\n# Use local sysroot.\nWEE8_BUILD_ARGS+=\" use_sysroot=false\"\n# Disable unused GLib2 dependency.\nWEE8_BUILD_ARGS+=\" use_glib=false\"\n# Expose debug symbols.\nWEE8_BUILD_ARGS+=\" v8_expose_symbols=true\"\n# Build monolithic library.\nWEE8_BUILD_ARGS+=\" is_component_build=false\"\nWEE8_BUILD_ARGS+=\" v8_enable_i18n_support=false\"\nWEE8_BUILD_ARGS+=\" v8_enable_gdbjit=false\"\nWEE8_BUILD_ARGS+=\" v8_use_external_startup_data=false\"\n# Disable read-only heap, since it's leaky and HEAPCHECK complains about it.\n# TODO(PiotrSikora): remove when fixed upstream.\nWEE8_BUILD_ARGS+=\" v8_enable_shared_ro_heap=false\"\n\n# Set target architecture.\nif [[ $${ARCH} == \"x86_64\" ]]; then\n  WEE8_BUILD_ARGS+=\" target_cpu=\\\"x64\\\"\"\nelif [[ $${ARCH} == \"aarch64\" || $${ARCH} == \"arm64\" ]]; then\n  WEE8_BUILD_ARGS+=\" target_cpu=\\\"arm64\\\"\"\nelif [[ $${ARCH} == \"ppc64le\" ]]; then\n  WEE8_BUILD_ARGS+=\" target_cpu=\\\"ppc64\\\"\"\nelif [[ $${ARCH} == \"s390x\" ]]; then\n  WEE8_BUILD_ARGS+=\" target_cpu=\\\"s390x\\\"\"\nfi\n\n# Select gn tool for the current platform.\nif [[ -f /etc/centos-release ]] && [[ $$(cat /etc/centos-release) =~ \"CentOS Linux release 7\" ]] && [[ -x \"$$(command -v gn)\" ]]; then\n  # Use system default on CentOS 7, as it has an old version of GLIBC which is otherwise incompatible with the prebuilt binary.\n  gn=$$(command -v gn)\nelif [[ $${PLATFORM} == \"Darwin-x86_64\" ]]; then\n  gn=buildtools/mac/gn\nelif [[ $${PLATFORM} == \"Darwin-arm64\" ]]; then\n  gn=buildtools/mac-arm64/gn\nelif [[ $${PLATFORM} == \"Linux-x86_64\" ]]; then\n  gn=buildtools/linux64/gn\nelif [[ $${PLATFORM} == \"Linux-aarch64\" ]]; then\n  gn=buildtools/linux-arm64/gn\nelif [[ $${PLATFORM} == MSYS_NT-*-x86_64 ]]; then\n  gn=buildtools/win/gn.exe\nelse\n  # Fallback to system default.\n  gn=$$(command -v gn)\nfi\n\n# Select ninja tool for the current platform.\nif [[ $${PLATFORM} == \"Darwin-x86_64\" ]]; then\n  ninja=third_party/depot_tools/ninja-mac\nelif [[ $${PLATFORM} == \"Linux-x86_64\" ]]; then\n  ninja=third_party/depot_tools/ninja-linux64\nelif [[ $${PLATFORM} == MSYS_NT-*-x86_64 ]]; then\n  ninja=third_party/depot_tools/ninja.exe\nelse\n  # Fallback to system default.\n  ninja=$$(command -v ninja)\nfi\n\n# Build wee8.\n\"$$gn\" gen out/wee8 --args=\"$$WEE8_BUILD_ARGS\"\n\"$$ninja\" -C out/wee8 wee8\n\n# Move compiled library to the expected destinations.\npopd\nmv $$ROOT/wee8/out/wee8/obj/libwee8.a $(execpath libwee8.a)\n"}
```

Comment:
@jwendell 🤔 just double checking, this change is to be expected correct?

Reply:
yeah, this is expected. This file is autogenerated by bazel upon processing the rules.

Reply:
Ah makes sense, thanks for clarifying. 👍